src/Microsoft.Identity.Client/Cache/Items/MsalRefreshTokenCacheItem.cs

```diff
@@ -72,6 +83,8 @@ internal static MsalRefreshTokenCacheItem FromJsonString(string json)
         internal static MsalRefreshTokenCacheItem FromJObject(JObject j)
         {
             var item = new MsalRefreshTokenCacheItem();
+            item.FamilyId = JsonUtils.ExtractExistingOrEmptyString(j, StorageJsonKeys.FamilyId);
```

Comment:
@MarkZuber - family_id is an optional field but I am 100% convinced that there are no implications for adding it. The current logic serializes it as an empty string if null, which is what we typically do with all our objects. Then again, I don't want this change (and the AppMetadata stuff) to break backwards-compat with MSAL.

Reply:
> The current logic serializes it as an empty string if null, which is what we typically do with all our objects.

FYI: Talked with @oldalton about this just now. The schema does not strictly define what a non-family value should be serialized; the objC implementation of test case chose to not serialize it at all. At the same time, the expectation is our SDKs should be flexible when deserializing `"foci": null`, `"foci": ""`, or the completely absence of the foci field as the same meaning as "not in a family".

Reply:
Thx @rayluo - this is my expectation as well! Next week I am in Redmond, let's sync up on this and on how the cache will look like.